t/basic.t

```diff
@@ -20,6 +20,7 @@ use lib "$FindBin::Bin/lib";
 use Test::Mojo;
 use Test::Warnings;
 use OpenQA::Test::Database;
+use OpenQA::Test::TimeLimit '6';
```

Comment:
I'm wondering about this weird usage. A simple test helper, e.g. called `set_timeout` didn't work?

Reply:
That would be boring :) As discussed in person I consider it a usual treat of perl modules to provide functionality on import and also in a configurable way. This is inspired by https://metacpan.org/pod/Time::Limit . Additional advantages I see with this approach:

* The limit is already effective from the time of import. Otherwise one would need to mix function calls with imports which is not that nice
* Saving some lines as the only functionality of this module is the module-wide time limit so there would be no need for further lines of code anyway

Reply:
"That would be boring" / "Saving some lines" - those are bad justifications. But "The limit is already effective from the time of import." is enough for me so why not. I think that explanation should go into a comment in `t/lib/OpenQA/Test/TimeLimit.pm`.